cvmfs/cvmfs_config

```diff
@@ -1833,15 +1833,23 @@ EOF
      # SIP) create directory in root `/`.
      # We decide to move the standard installation directory on
      # `/Users/shared/cvmfs` instead of `/cvmfs`
+     # In order to keep the standard CVMFS mountpoint, we decide to implement
+     # firmlinks from `/cvmfs` to `/Users/shared/cvmfs`
      if sw_vers | grep 10.15; then
```

Comment:
Ah, sorry, I forgot: this should check for >= 10.15, otherwise it will break again with 10.16...

Reply:
Addressed in #2450